app/Models/Status.php

```diff
@@ -63,6 +63,22 @@ public function getFavoritedAttribute(): ?bool {
         return $this->likes->contains('user_id', Auth::id());
     }
 
+    public function getIsLikableProperty(): bool {
+        return $this->isLikable();
+    }
+
+    public function isLikable(): bool {
+        if (!$this->user->likes_enabled) {
+            return false;
+        }
+
+        if (!Auth::check()) {
+            return true;
+        }
+
+        return Auth::user()->likes_enabled;
+    }
```

Comment:
Why do you create two redundant methods? Isn't the model property enough?

Reply:
(Maybe it would be better to move this logic to an Policy.)

Reply:
Kann man schon machen, ich hab nur keine Ahnung von Policies. Aber tu dir keinen Zwang an c:

> Isn't the model property enough?

Das property ist manchmal nicht evaluiert, wenn ich das im Code abgefragt habe, sondern war dann auf `null`, darum lieber so. Aber kann auch Jannik-Fettfingering sein.

Reply:
Okay, ich probiere mich mal!